src/utils/wallet.js

```diff
@@ -451,237 +421,25 @@ const wallet = {
-      // then we don't know if we've cleaned up the wallet data in the storage
-      // If it's fromXpriv, then we can't clean access data because we need that
-      oldWalletUtil.cleanLoadedData({ cleanAccessData: !fromXpriv});
-
-      let xpriv = null;
-
-      if (fromXpriv) {
-        xpriv = oldWalletUtil.getXprivKey(pin);
-      }
-
-      connection = new Connection({
-        network: network.name,
-        servers: [helpers.getServerURL()],
-      });
-
-      const beforeReloadCallback = () => {
-        store.dispatch(loadingAddresses(true));
-      };
-
-      const walletConfig = {
-        seed: words,
-        xpriv,
-        xpub,
-        store: STORE,
-        passphrase,
-        connection,
-        beforeReloadCallback,
-      };
-
-      wallet = new HathorWallet(walletConfig);
-    }
-
-    store.dispatch(setWallet(wallet));
-
-    // TODO should we save server info?
-    //store.dispatch(setServerInfo(serverInfo));
-    wallet.on('state', async (state) => {
-      if (state === HathorWallet.ERROR) {
-        // ERROR
-        // TODO Should we show an error screen and try to restart the wallet?
-      } else if (wallet.isReady()) {
-        // READY
-        const { tokensHistory, tokensBalance, tokens } = await this.fetchWalletData(wallet);
-        store.dispatch(loadWalletSuccess(tokensHistory, tokensBalance, tokens));
-      }
-    });
-
-    wallet.on('new-tx', async (tx) => {
-      let message = '';
-      if (helpers.isBlock(tx)) {
-        message = 'You\'ve found a new block! Click to open it.';
-      } else {
-        message = 'You\'ve received a new transaction! Click to open it.'
-      }
-
-      const notification = this.sendNotification(message);
-
-      // Set the notification click, in case we have sent one
-      if (notification !== undefined) {
-        notification.onclick = () => {
-          routerHistory.push(`/transaction/${tx.tx_id}/`);
-        }
-      }
-
-      this.fetchNewTxTokenBalance(wallet, tx).then(async (updatedBalanceMap) => {
-        if (updatedBalanceMap) {
-          this.handlePartialUpdate(wallet, updatedBalanceMap);
-
-          // If we are here, we have already fetched new addresses, so it's safe to
-          // call getCurrentAddress to make sure we display a new address to the user
-          const currentAddress = wallet.getCurrentAddress();
-          this.updateSharedAddressRedux(currentAddress.address, currentAddress.index);
-        }
-      });
-    });
-
-    wallet.on('update-tx', async (tx) => {
-      // `update-tx` event is not yet being emitted from the wallet-service facade as it is not implemented,
-      // we should ignore this message for now to prevent errors when we start emitting it in the future
-      if (useWalletService) {
-        return;
-      }
-      const balances = await wallet.getTxBalance(tx, { includeAuthorities: true });
-      const updatedBalanceMap = await this.fetchNewTxTokenBalance(wallet, tx);
```

Comment:
Can we remove this `fetchNewTxTokenBalance` method?

Reply:
Yes we can, done in 54e0653e27af8005ff919e0825c78d3110e68015